examples/custom-config/postgres-ha.yaml

```diff
@@ -0,0 +1,18 @@
+---
+bootstrap:
+  dcs:
+    postgresql:
+      parameters:
+        log_directory: pglogs
+        log_min_duration_statement: 0
+        log_statement: none
+        max_wal_senders: 6
+        shared_preload_libraries: pgaudit.so
+postgresql:  
+  pg_hba:
+    - local all postgres peer
+    - local all crunchyadm peer
+    - host replication primaryuser 0.0.0.0/0 md5
+    - host all primaryuser 0.0.0.0/0 reject
+    - host all testuser1 0.0.0.0/0 md5
+    - host all testuser2 0.0.0.0/0 md5
```

Comment:
In our current setup, it would be fine to do

```yaml
- host all postgres 0.0.0.0/0md5
```

also because without that, we block the `postgres` user from being able to log in and take the actions that are available in `pgo [create|update|delete] user` user.

Reply:
This was added in a subsequent commit: https://github.com/CrunchyData/postgres-operator/pull/1184/commits/7549466771ffdaff8d6f644d46332049da504d0f